### PR TITLE
make the value "ignore" available

### PR DESCRIPTION
**Pull request by guy-maurel**
for the options:
sp_after_ptr_star, sp_after_ptr_star_func,
sp_after_ptr_star_trailing, sp_after_ptr_star_qualifier,
sp_before_ptr_star, sp_before_ptr_star_func,
sp_before_ptr_star_trailing, sp_before_unnamed_ptr_star,
sp_between_ptr_star, sp_ptr_star_paren

### CODE REVIEW

File: src/options.h [hunk 1]
```diff
@@ -262,6 +262,7 @@ extern Option<iarf_e>
 sp_before_unnamed_ptr_star;
 
 // Add or remove space between pointer stars '*'.
+// as in 'int ***a;'.
```
Comment (mwoehlke-kitware) [inline src/options.h:265]:
```suggestion
// Add or remove space between pointer stars '*', as in 'int ***a;'.
```

File: src/options.h [hunk 2]
```diff
@@ -286,8 +287,13 @@ sp_after_ptr_star_qualifier;
 extern Option<iarf_e>
 sp_after_ptr_star_func;
 
+// Add or remove space after a pointer star '*', in the trailing return
+// a function prototype or function definition.
```
Comment (mwoehlke-kitware) [inline src/options.h:291]:
```suggestion
// Add or remove space after a pointer star '*' in the trailing return of a
// function prototype or function definition.
```

File: src/options.h [hunk 3]
```diff
@@ -296,6 +302,11 @@ sp_ptr_star_paren;
 extern Option<iarf_e>
 sp_before_ptr_star_func;
 
+// Add or remove space before a pointer star '*', in the trailing return
+// a function prototype or function definition.
```
Comment (mwoehlke-kitware) [inline src/options.h:306]:
```suggestion
// Add or remove space before a pointer star '*' in the trailing return of a
// function prototype or function definition.
```

File: src/space.cpp [hunk 4]
```diff
@@ -120,6 +120,33 @@ const no_space_table_t no_space_table[] =
 };
 
 
+bool token_is_within_Trailing_Return(chunk_t *pc)
```
Comment (mwoehlke-kitware) [inline src/space.cpp:123]: BTW, how come some words are capitalized?

**Review by mwoehlke-kitware — COMMENTED** [review]
Seems like a good change, thanks! I left some suggestions for improving the consistency and grammar of the documentation.

File: src/space.cpp (hunk 4 above)
Comment (gmaurel) [reply src/space.cpp]: Do we have a rule for that? Where?

**Review by gmaurel — APPROVED** [review]

**Pull request merged**